<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ibm360ProcessorEditor</class>
 <widget class="QWidget" name="Ibm360ProcessorEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="_clockFrequencyLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Clock frequency:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="_clockFrequencyUnitComboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="_clockFrequencyLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="_featuresGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>231</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Features</string>
   </property>
   <widget class="QCheckBox" name="_fetchProtectionCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Fetch protection</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="_storeProtectionCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Store protection</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="_decimalCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Decimal</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="_floatingPointCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Floating point</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="_byteOrientedOperandCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Byte-oriented operand</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="_timerCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Timer</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="_directControlCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Direct control</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_clockFrequencyLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Ibm360ProcessorEditor</receiver>
   <slot>_clockFrequencyLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_clockFrequencyUnitComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Ibm360ProcessorEditor</receiver>
   <slot>_clockFrequencyUnitComboBoxCurrentIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_fetchProtectionCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>Ibm360ProcessorEditor</receiver>
   <slot>_fetchProtectionCheckBoxClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_storeProtectionCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>Ibm360ProcessorEditor</receiver>
   <slot>_storeProtectionCheckBoxClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_decimalCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>Ibm360ProcessorEditor</receiver>
   <slot>_decimalCheckBoxClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_floatingPointCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>Ibm360ProcessorEditor</receiver>
   <slot>_floatingPointCheckBoxClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_byteOrientedOperandCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>Ibm360ProcessorEditor</receiver>
   <slot>_byteOrientedOperandCheckBoxClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_timerCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>Ibm360ProcessorEditor</receiver>
   <slot>_timerCheckBoxClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_directControlCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>Ibm360ProcessorEditor</receiver>
   <slot>_directControlCheckBoxClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_clockFrequencyLineEditTextChanged(QString)</slot>
  <slot>_clockFrequencyUnitComboBoxCurrentIndexChanged(int)</slot>
  <slot>_fetchProtectionCheckBoxClicked()</slot>
  <slot>_storeProtectionCheckBoxClicked()</slot>
  <slot>_decimalCheckBoxClicked()</slot>
  <slot>_floatingPointCheckBoxClicked()</slot>
  <slot>_byteOrientedOperandCheckBoxClicked()</slot>
  <slot>_timerCheckBoxClicked()</slot>
  <slot>_directControlCheckBoxClicked()</slot>
 </slots>
</ui>
